src/components/atoms/modal/modal.tsx

```diff
@@ -21,10 +23,12 @@ export const Modal = ({ modalTitle, modalContent }: Props): JSX.Element => {
     <button className="modal-button" onClick={() => setShowModal(!showModal)}>Modal Link</button>
```

Comment:
Same comments as before about buttons, but for Modal, I'm assuming we don't want to be including a button in the component, rather having the ability to open and close modals based on an external event of any description? Again, is this planned for later still?